metagraph/src/graph/alignment/aligner_helper.cpp

```diff
@@ -44,7 +44,8 @@ bool DPTable<NodeType>::add_seed(const Alignment<NodeType> &seed,
         table_init.gap_scores[start_pos] = std::max(
             last_op == Cigar::DELETION
                 ? table_init.scores[start_pos]
-                : table_init.scores[start_pos] - last_char_score + config.gap_opening_penalty + config.gap_opening_penalty,
+                : table_init.scores[start_pos] - last_char_score
+                    + config.gap_opening_penalty + config.gap_opening_penalty,
```

Comment:
```suggestion
                : table_init.scores[start_pos] - last_char_score
                                               + config.gap_opening_penalty
                                               + config.gap_opening_penalty,
```
way easier to read

Reply:
is this on purpose you add `config.gap_opening_penalty` twice?

Reply:
I'll add a comment to describe this. This array is used to store the best score achieved if the last operation was a deletion. So, I'm initializing this to the value that you would get if you performed an insertion and a deletion instead of a character match.